tasks/handle_error_exit.sh

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2015-present, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+
+# print error messages to stderr
+# the cleanup function is optionally defined in caller script
+function handle_error {
```

Comment:
I would prefer we keep these duplicated in all files. We’re not shell experts so it’s easier if we don’t `source` any files. Duplication is not a problem for us.